python/format_code.sh

```diff
@@ -8,8 +8,12 @@ fi
 
 PYTHON_PACKAGES=(covid_graphs covid_web)
 
-echo "Formatting imports (isort)"
-isort --apply --recursive $PYTHON_PACKAGES
-
-echo "Formatting code (black)"
-black $PYTHON_PACKAGES
+for PYTHON_PACKAGE in "${PYTHON_PACKAGES[@]}"
```

Comment:
Paging @miskosz , so that he takes a look.

Reply:
Created separate issue #110